### DISCUSSION TITLE
What’s the best way to have separate tours?

### DISCUSSION THREAD

**Discussion by JackyGohSG** [gilbarbara/react-joyride#1118] [General]
I already have a basic multi route tour in my app. I want a more in depth tour for each separate section. What is the best way to do that?

**Comment by gilbarbara** [comment]
Hey @JackyGohSG 

It's hard to advise without more information, but if you already have a multi-route tour, you could stop it when you reach a specific route by using the callback and start the local tour when you detect the global one has paused. When the local one has finished/skipped, restart the global one.
